src/server.js

```diff
@@ -1,10 +1,16 @@
+'use strict'; // eslint-disable-line
 const express = require('express');
 const bodyParser = require('body-parser');
 const morgan = require('morgan');
 const winston = require('winston');
 
-const options = { logger: winston };
 const config = require('../config');
+
+const options = {
+  config,
```

Comment:
Any particular reason why we're sharing config as a global on the application object?  (This is usually how I do it, just curious about the change)